### ISSUE TITLE
cairo: version bump

### ISSUE THREAD

**Issue by luzpaz** [AOSC-Dev/aosc-os-abbs#1639]
https://cairographics.org/news/cairo-1.16.0/  
https://cairographics.org/releases/cairo-1.16.0.tar.xz

**Comment by eatradish** [comment]
oh
```
root@saki-exp-11007b [ cairo@explosive ] ! apt-cache rdepends cairo
cairo
Reverse Depends:
  harfbuzz
  cairocffi
  vlc
  harfbuzz
  harfbuzz
  vlc
  ruby-cairo
  harfbuzz
  gtk-3
  gjs
  gst-plugins-good-1-0
  vlc
  kicad
  tesseract
  harfbuzz
  vlc
  ruby-cairo
  gtk-3
  fcitx5
  cjs
  awesome
  pango
  gjs
  synfig
  gst-plugins-good-1-0
  astrometry.net
  retro-gtk
  kicad
  x11vnc
  vlc
  tesseract
  ruby-cairo
  poppler
  kicad
  harfbuzz
  fcitx5
  cairocffi
  cairocffi
  cairocffi
  vlc
  vlc
  libgxps
  x11vnc
  x11vnc
  vlc
  vlc
  vlc
  vlc
  vlc
  vlc
  tesseract
  synfig
  synfig
  scribus
  ruby-cairo
  ruby-cairo
  retro-gtk
  pycairo
  pycairo
  poppler
  poppler
  poppler
  poppler
  poppler
  pango
  libgdiplus
  matchbox-keyboard
  harfbuzz
  harfbuzz
  gtk-3
  gtk-2
  gst-plugins-good-1-0
  gst-plugins-good-1-0
  gnuplot
  gnuplot
  gnash
  gnash
  gjs
  cjs
  astrometry.net
  kicad
  kicad
  gtk-3
  vlc
  vlc
  gjs
  pango
  ruby-cairo
  ruby-cairo
  gst-plugins-good-1-0
  gst-plugins-good-1-0
  gst-plugins-good-1-0
  harfbuzz
  harfbuzz
  harfbuzz
  synfig
  astrometry.net
  astrometry.net
  poppler
  poppler
  retro-gtk
  pango
  poppler
  gst-plugins-good-1-0
  x11vnc
  wesnoth
  vlc
  tesseract
  scribus
  pycairo
  poppler
  perl-cairo
  memphis
  cjs
  libgdiplus
  intel-gpu-tools
  i3lock
  gtk-2
  gst-plugins-good-0-10
  gnustep-back
  gnuplot
  gnash
  awesome
  cairomm
  ruby-cairo
  kicad
  gtk-3
  gjs
  pango
  libgxps
  gst-plugins-good-1-0
  harfbuzz
  synfig
  astrometry.net
  retro-gtk
  libchamplain
  ntk
```

**Comment by eatradish** [comment]
ok, now cairo 1.16.0 in explosive, But there are a lot of packages that need rebuild.........qaq

**Comment by eatradish** [comment]
Ah, I looked at the list of files again, but I don’t need to rebuild...

**Comment by MingcongBai** [comment]
Addressed with b9eb2df644690e45d1d90df076716cd232c8f9e1. Closing.
